platform/config/config-reader-yaml/src/test/resources/proxy.yml

```diff
@@ -0,0 +1,26 @@
+---
```

Comment:
This file can be removed also as it is used in the test that can be removed.

Reply:
☝️